src/Maker/MakeCrud.php

```diff
@@ -95,6 +100,10 @@ public function interact(InputInterface $input, ConsoleStyle $io, Command $comma
             sprintf('Choose a name for your controller class (e.g. <fg=yellow>%s</>)', $defaultControllerClass),
             $defaultControllerClass
         );
+
+        if (class_exists(CssSelectorConverter::class)) {
+            $this->generateTests = $io->confirm('Do you want to generate tests for the controller? [Experimental]', false);
+        }
```

Comment:
What would happen if we didn't check for the class? And just always asked the question? If they ran the tests, obviously they wouldn't work, but would it be obvious that they just need to install some stuff? I'm trying to limit the number of conditionals and paths we have.